modules/react-components/src/components/modal/confirmation-modal/confirmation-modal.tsx

```diff
@@ -159,6 +164,16 @@ export const ConfirmationModal: FunctionComponent<ConfirmationModalPropsInterfac
         onPrimaryActionClick(e);
     };
 
+    /**
+     * Handler for the enter key press down.
+     */
+    const handleKeyDown = (e: KeyboardEvent) => {
+        if (e.key === "Enter" && confirmed) {
+            setAssertionInput("");        
+            onEnterKeyPressConfirm(e);
```

Comment:
Let's trigger the primary action when the `enter` key is pressed. This way we will not have to refactor everywhere where this modal is used.

If you think that we should give the ability to override this behaviour, you can do something like below.

```
if (onEnterKeyPressConfirm) {
     onEnterKeyPressConfirm(e);
     return;
}

onPrimaryActionClick(e);
```
```suggestion
            onPrimaryActionClick(e);
```

Reply:
The proposed change added with additional param in method